src/components/menu/menu-item/menu-item.component.js

```diff
@@ -31,6 +30,7 @@ const MenuItem = ({
   icon,
   selected,
   onKeyDown,
+  onFocus,
```

Comment:
what does this do?

Reply:
Absolutely nothing. Think I added it when I was debugging and never removed it. Cheers for spotting it.